requirements.txt

```diff
@@ -1,5 +1,5 @@
 EasyDict==1.7
-opencv-python==3.4.1.15
+opencv-python
```

Comment:
why delete the version?

Reply:
I could not find this version of opencv-python on my local machine using conda python.